examples/Makefile.am

```diff
@@ -19,11 +19,16 @@ UPIPEOSX_LIBS = $(top_builddir)/lib/upipe-osx/libupipe_osx.la
 UPIPEDVBCSA_LIBS = $(top_builddir)/lib/upipe-dvbcsa/libupipe_dvbcsa.la
 UPIPEDVB_LIBS = $(top_builddir)/lib/upipe-dvb/libupipe_dvb.la
 UPIPEBEARSSL_LIBS = $(top_builddir)/lib/upipe-bearssl/libupipe_bearssl.la
+UPIPESRT_LIBS = $(top_builddir)/lib/upipe-srt/libupipe_srt.la
 
 noinst_PROGRAMS = 
 
 fec_LDADD = $(LDADD) $(UPUMPEV_LIBS) $(UPIPEMODULES_LIBS) $(UPIPETS_LIBS)
 rist_rx_LDADD = $(LDADD) $(UPUMPEV_LIBS) $(UPIPEMODULES_LIBS) $(UPIPEFILTERS_LIBS)
+srt_rx_LDADD = $(LDADD) $(UPUMPEV_LIBS) $(UPIPEMODULES_LIBS) $(UPIPEFILTERS_LIBS) $(UPIPESRT_LIBS)
+srt_rx_CFLAGS = $(AM_CFLAGS) $(BITSTREAM_CFLAGS)
+srt_tx_LDADD = $(LDADD) $(UPUMPEV_LIBS) $(UPIPEMODULES_LIBS) $(UPIPEFILTERS_LIBS) $(UPIPESRT_LIBS)
+srt_tx_CFLAGS = $(AM_CFLAGS) $(BITSTREAM_CFLAGS)
```

Comment:
I don't see any bitstream usage in `srt_rx` and `srt_tx`? Same for upipe-filters?